routes/boxHistory.js

```diff
@@ -80,19 +81,34 @@ boxHistoryRouter.get('/:transactionID', async (req, res) => {
   }
 });
 
+// Gets a box's history
+boxHistoryRouter.get('/history/:boxID', async (req, res) => {
+  const { boxID } = req.params;
+  try {
+    const history = await getHistoryOfBox(boxID);
+    res.status(200).send(history);
+  } catch (err) {
+    res.status(400).send(err.message);
```

Comment:
Same here - replace 400 with 500 error. 400 is a client-side error while 500 is server-side error